test/libsolidity/semanticTests/isoltestTesting/bytecode.sol

```diff
@@ -0,0 +1,18 @@
+contract SmokeTest {
+}
+// ====
+// compileViaYul: also
+// ----
+// constructor()
+// bytecode: 0x00112233 -> 0
+// bytecode: 0x60806040526 -> 1
+// bytecode: 0x806040526 -> 1
+// bytecode: 0x6040526 -> 1
+// bytecode: 0x5040526 -> 0
+// bytecode: "(60).*" -> 1
```

Comment:
How does isoltest suggestion look like when the expectation fails? Won't it try to reformat `"(60).*"` as a hex number or the other way around?

Reply:
OK, based on what I saw in the event PR I guess it would only do it the other way around - i.e. reformat as string when has has only printable characters.